types/logger.d.ts

```diff
@@ -1,5 +1,7 @@
 import { FastifyError } from './error'
 import { RawServerBase, RawServerDefault, RawRequestDefaultExpression, RawReplyDefaultExpression } from './utils'
+import { FastifyRequest } from './request'
+import { LoggerOptions as PinoLoggerOptions } from 'pino'
```

Comment:
please see previous comment about intentionally omitting pino types

Reply:
Thanks for the heads up! I've removed the pino types.